app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated")
+        if (visitor.get("vaccine").get("expiration_date")
+                < datetime.date.today()):
```

Comment:
Vaccine is already present, if not, the first error will be risen. So you can just leave as it was before.
If the expiration date is not present, None will be returned and you will get `TypeError: '<' not supported between instances of 'NoneType' and 'datetime.datetime'`